installer/bundle/bundle_skel.sh

```diff
@@ -293,26 +293,17 @@ pkg_upd_list() {
     fi
 }
 
-get_arch()
-{
-    if [ $(uname -m) = 'x86_64' ]; then
-        echo "x64"
-    else
-        echo "x86"
-    fi
-}
-
 compare_arch()
 {
     #check if the user is trying to install the correct bundle (x64 vs. x86) 
     echo "Checking host architecture ..."
-    AR=$(get_arch)
+    HOST_ARCH="$(getconf LONG_BIT)" 
```

Comment:
This returns 32 for x86 architecture; my suggestion would be to keep the get_arch() method and echo this refactored result out like before.